tests/RobotFramework/tests/tedge_connect/tedge_connet_mqtt_service.robot

```diff
@@ -0,0 +1,55 @@
+*** Settings ***
+Resource            ../../resources/common.resource
+Library             DateTime
+Library             Cumulocity
+Library             ThinEdgeIO
+
+Test Teardown       Get Logs
+
+
+*** Test Cases ***
+Connect to Cumulocity MQTT Service endpoint
+    ${DEVICE_SN}=    Setup    connect=${False}
+    Execute Command    tedge config set c8y.tenant_id t37070943
+    Execute Command    tedge config set c8y.mqtt_service.enabled true
+    Execute Command    tedge config set c8y.mqtt_service.topics 'sub/topic,demo/topic'
+    Execute Command    tedge connect c8y
```

Comment:
How can I check that the connection to the MQTT service is actually established?

- Neither `tedge connect c8y` nor `tedge connect c8y --test` tells a word about the MQTT service and status.
- I tried on purpose to break the config, no errors are returned by `tedge connect`.

Good. The bridge status published on `te/device/main/service/mosquitto-c8y-mqtt-bridge/status/health` is working and reports broken as well as correct settings.

=> I would add a test assertion that a status message of `1` is received on this topic.